tests/components/lutron_caseta/test_init.py

```diff
@@ -0,0 +1,57 @@
+"""Tests for the Lutron Caseta integration."""
+
+from unittest.mock import patch
+
+import pytest
+
+from homeassistant.config_entries import ConfigEntryState
+from homeassistant.core import HomeAssistant
+
+from . import MockBridge, async_setup_integration, make_mock_entry
+
+
+async def test_timeout_during_connect(
```

Comment:
can merge this test and the one following, by parameterizing `timeout_during_connect` and `timeout_during_configure` and the expected log message?
we could also patch both `CONNECT_TIMEOUT` and `CONFIGURE_TIMEOUT` in decorators to make it cleaner

Reply:
done in 11c05d9013a